### PR TITLE
Issue 6226. Remove property chroot declaration from FileTransfer

### LINKED ISSUE #6226: Regression on Windows: Backdrop 1.26.0 release GUI install module Error: File Transfer failed
## Description of the bug

After installing upgrade to Backdrop 1.26.0 from 1.25.1 used the GUI interface to install a new module. This failed with Error.

```
Error installing Porter Stemmer. Error installing / updating. Error: File Transfer failed, reason: .../backdrop-demo/modules is outside of the .../backdrop-demo
```

## Steps To Reproduce

To reproduce the behavior:
1. Go to 'admin/modules/install'
2. Search for module to install.
3. Click on 'Add to installation queue'
4. Click on 'Install' in right sidebar
5. Proceed to install.
6. See Report of error.

## Actual behavior

Error.
```
Error installing Metatag. Error installing / updating. Error: File Transfer failed, reason: .../backdrop-demo/modules is outside of the .../backdrop-demo
```

## Expected behavior

Normally, and previously with Backdrop 1.25.1 the installation would proceed without issue.

## Additional information

Add any other information that could help, such as:
- Backdrop CMS version: 1.26.0
- Web server and its version: XAMPP Apache 2.4.56
- PHP version: 8.1.17
- Database sever (MySQL or MariaDB?) and its version: 10.4.28-MariaDB
- Operating System and its version: Windows 10
- Browser(s) and their versions: Firefox

### PR DESCRIPTION
**Pull request by argiepiano**
Fixes https://github.com/backdrop/backdrop-issues/issues/6226

### CODE REVIEW

**Comment by muranod** [comment]
Thanks! This fixed the issue for me. Thanks too, for making the raw files available. (Windows 11 / XAMPP)

**Comment by argiepiano** [comment]
> Thanks! This fixed the issue for me. Thanks too, for making the raw files available. (Windows 11 / XAMPP)

@muranod it'd be helpful if you post your comment also in the main queue for this issue:  https://github.com/backdrop/backdrop-issues/issues/6226

File: core/includes/filetransfer/filetransfer.inc [hunk 1]
```diff
@@ -11,31 +11,76 @@
...
 
   /**
+   * Full path to directory where file-transfer is restricted to.
+   *
    * @var string
    */
   protected $jail;
 
+    /**
+   * Path to connection chroot.
```
Comment (quicksketch) [inline core/includes/filetransfer/filetransfer.inc]:
```suggestion
  /**
   * Path to connection chroot.
```

File: core/includes/filetransfer/filetransfer.inc [hunk 2]
```diff
@@ -61,19 +106,60 @@ abstract class FileTransfer {
...
 
     if ($name == 'chroot') {
       $this->setChroot();
-      return $this->chroot;
+      return $this->chrootPath;
+    }
+  }
+
+  /**
+   * {@inheritdoc}
```
Comment (quicksketch) [inline core/includes/filetransfer/filetransfer.inc]: `{@inheritdoc}` is used when there's a parent class that has this same method. But this `abstract class FileTransfer` does not extend any other classes. Could you document this and the other new magic methods?

**Comment by quicksketch** [comment]
Merged into 1.x and 1.26.x. Thank you @argiepiano!!

**Pull request merged**